src/pages/TCY/components/TCYHeader.tsx

```diff
@@ -0,0 +1,72 @@
+import { Flex, Heading } from '@chakra-ui/react'
+import { arbitrumChainId } from '@shapeshiftoss/caip'
+import { useCallback, useMemo } from 'react'
+import { useTranslate } from 'react-polyglot'
+import { useNavigate } from 'react-router-dom'
+
+import { AccountDropdown } from '@/components/AccountDropdown/AccountDropdown'
+import { Display } from '@/components/Display'
+import { InlineCopyButton } from '@/components/InlineCopyButton'
+import { PageBackButton, PageHeader } from '@/components/Layout/Header/PageHeader'
+import { SEO } from '@/components/Layout/Seo'
+import { Text } from '@/components/Text'
+import { selectAccountIdsByChainIdFilter } from '@/state/slices/portfolioSlice/selectors'
+import { useAppSelector } from '@/state/store'
+
+export const TCYHeader = () => {
+  const translate = useTranslate()
+  const navigate = useNavigate()
+
+  const handleBack = useCallback(() => {
+    navigate('/explore')
+  }, [navigate])
+
+  const accountIds = useAppSelector(state =>
+    selectAccountIdsByChainIdFilter(state, { chainId: arbitrumChainId }),
+  )
+
+  const handleChange = useCallback(() => {}, [])
+
+  const activeAccountDropdown = useMemo(() => {
+    if (accountIds.length <= 1) return null
+
+    return (
+      <Flex alignItems='center' gap={2}>
+        <Text translation='common.activeAccount' fontWeight='medium' />
+
+        <InlineCopyButton value={''} />
+        <AccountDropdown
+          defaultAccountId={''}
+          assetId={''}
+          onChange={handleChange}
+          // dis already memoized
+          // eslint-disable-next-line react-memo/require-usememo
+          buttonProps={{ variant: 'solid', width: 'full' }}
```

Comment:
I know this was copied from another part of the codebase, but I don't think disabling the `react-memo/require-usememo` is correct in this context (or the existing instance).

Having an object inside the memoization context means we unnecessarily recreate that object on every render of `activeAccountDropdown` (whenever `accountIds.length` or `handleChange` changes).